src/main/java/org/apache/sling/junit/impl/servlet/TestLogServlet.java

```diff
@@ -57,11 +61,53 @@
 import ch.qos.logback.core.Layout;
 import ch.qos.logback.core.read.CyclicBufferAppender;
 
-@Component(immediate=true, metatype=true,
-        label = "Apache Sling Test Log Collector",
-        description = "Servlet that exposes logs collected for a particular test execution"
+@Component(
+    service = Servlet.class,
+    immediate = true,
+    configurationPolicy = ConfigurationPolicy.OPTIONAL,
+    property = {
```

Comment:
no need for that as you use property configuration type

Reply:
Fixed in the latest change.